napalm/base/models.py

```diff
@@ -224,15 +224,31 @@
 NTPPeerDict = TypedDict(
     "NTPPeerDict",
     {
-        # will populate it in the future wit potential keys
+        "address": str,
```

Comment:
Adding the same keys here was a quick and dirty fix to avoid refactoring nxos's mypy typing for same.  I may revisit.